experimental/sdk/webpubsub-socketio-extension/src/SIO/components/web-pubsub-adapter.ts

```diff
@@ -245,9 +283,10 @@ export class WebPubSubAdapterInternal extends NativeInMemoryAdapter {
    * @param close - whether to close the underlying connection
    */
   public async disconnectSockets(opts: BroadcastOptions, close: boolean): Promise<void> {
-    debug(`disconnectSockets, start, opts = ${JSON.stringify(opts)}, close = ${close}`);
+    debug(`disconnectSockets, start, opts = ${toOptionsString(opts)}, close = ${close}`);
     await this.broadcast({ type: SioPacketType.DISCONNECT, nsp: this.nsp.name, data: { close } } as SioPacket, opts);
-    debug(`disconnectSockets, finish, opts = ${JSON.stringify(opts)}, close = ${close}`);
+    super.disconnectSockets(opts, close);
```

Comment:
Can `super.disconnectSockets(opts, close)` cause some duplicated messages?

Reply:
After discussion, after broadcasting DISCONNECT, server should do nothing. Service will send DISCONNECT or CLOSE to server later.